go.mod

```diff
@@ -14,22 +14,22 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/cenkalti/backoff/v4 v4.3.0
 	github.com/fatih/color v1.18.0
-	github.com/google/go-cmp v0.6.0
+	github.com/google/go-cmp v0.7.0
 	github.com/google/go-querystring v1.1.0
 	github.com/google/uuid v1.6.0
 	github.com/hashicorp/go-memdb v1.3.4
 	github.com/hashicorp/go-retryablehttp v0.7.7
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/kong/deck v1.44.2
-	github.com/kong/go-kong v0.63.0
+	github.com/kong/go-kong v0.63.1-0.20250311100117-3fb9f0059d0c
```

Comment:
This will be updated once go-kong is released.
https://github.com/Kong/go-kong/pull/507